undefined reference to `openpty'

On a fresh image build today and a fresh QGIS master:

```
[1/2] Building C object CMakeFiles/cmTC_69d3d.dir/CheckFunctionExists.c.o
[2/2] Linking C executable cmTC_69d3d
FAILED: cmTC_69d3d 
: && /usr/bin/cc -g -O2 -fdebug-prefix-map=/qgis/QGIS=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -O2 -Wdate-time -D_FORTIFY_SOURCE=2 -DCHECK_FUNCTION_EXISTS=openpty  -Wl,-z,relro -Wl,-z,now   -rdynamic CMakeFiles/cmTC_69d3d.dir/CheckFunctionExists.c.o  -o cmTC_69d3d   && :
/usr/bin/ld: CMakeFiles/cmTC_69d3d.dir/CheckFunctionExists.c.o: in function `main':
/usr/share/cmake-3.13/Modules/CheckFunctionExists.c:17: undefined reference to `openpty'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.


dh_auto_configure: cd debian/build && cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -G Ninja -DBUILDNAME=buster-amd64 -DCMAKE_VERBOSE_MAKEFILE=1 -DCMAKE_INSTALL_PREFIX=/usr -DBINDINGS_GLOBAL_INSTALL=TRUE -DPEDANTIC=TRUE -DSERVER_SKIP_ECW=TRUE -DQGIS_CGIBIN_SUBDIR=/usr/lib/cgi-bin -DWITH_APIDOC=TRUE -DGENERATE_QHP=TRUE -DWITH_CUSTOM_WIDGETS=TRUE -DWITH_GLOBE=FALSE -DWITH_SERVER=TRUE -DWITH_SERVER_PLUGINS=TRUE -DWITH_QWTPOLAR=FALSE -DQT_PLUGINS_DIR=usr/lib/x86_64-linux-gnu/qt5/plugins -DPYTHON_LIBRARY=/usr/lib/python3.7/config-3.7m-x86_64-linux-gnu/libpython3.7m.so -DDOXYGEN_ON_DEMAND=TRUE -DWITH_GRASS=TRUE -DWITH_GRASS7=TRUE -DGRASS_PREFIX7=/usr/lib/grass76 -DWITH_QSPATIALITE=TRUE -DPOSTGRES_LIBRARY=/usr/lib/x86_64-linux-gnu/libpq.so -DWITH_3D=TRUE -DGEOS_LIBRARY=/usr/lib/x86_64-linux-gnu/libgeos_c.so -DCMAKE_PREFIX_PATH=/qgis/QGIS/external/qt3dextra-headers/cmake -DQT5_3DEXTRA_INCLUDE_DIR=/qgis/QGIS/external/qt3dextra-headers -DQT5_3DEXTRA_LIBRARY=/usr/lib/x86_64-linux-gnu/libQt53DExtras.so -DSPATIALINDEX_LIBRARY=/usr/lib/x86_64-linux-gnu/libspatialindex.so -DENABLE_TESTS=FALSE ../.. returned exit code 1
make[2]: *** [debian/rules:229: override_dh_auto_configure] Error 2
make[2]: Leaving directory '/qgis/QGIS'
make[1]: *** [debian/rules:221: build-arch] Error 2
make[1]: Leaving directory '/qgis/QGIS'
make: *** [debian/rules:221: build] Error 2
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

--- a/qgis-build/Dockerfile
+++ b/qgis-build/Dockerfile
@@ -86,6 +86,11 @@ RUN apt-get update \
         qtpositioning5-dev \
         qttools5-dev \
         qttools5-dev-tools \
+        qt3d5-dev \
+        qt3d-assimpsceneimport-plugin \
+        qt3d-defaultgeometryloader-plugin \
+        qt3d-gltfsceneio-plugin \
+        qt3d-scene2d-plugin
         saga \
         spawn-fcgi \
         txt2tags \

